app/services/keyword_import.rb

```diff
@@ -14,20 +14,14 @@ def call(file)
     row_number = 0
     CSV.foreach(file) do |row|
       row_number += 1
-      begin
-        keyword = create_instance(row)
-        if keyword.valid?
-          results << keyword
-        else
-          errors << {
-            row: row_number,
-            message: keyword.errors.full_messages.to_sentence,
-          }
-        end
-      rescue ActiveRecord::RecordNotFound, ArgumentError => error
+
+      keyword = create_instance(row)
```

Comment:
this is not creating yet, it's instanciating

Reply:
@caherrerapa ,I actually meant creating an object instance. As it seems to be misleading, I renamed it to `instantiate_keyword_instance` in https://github.com/rajibds/google-search-from-csv/pull/38/commits/dc400347751e66ac04732db2f4a3381aff7a7b7c, Thank you!